ZAPD/Main.cpp

```diff
@@ -429,6 +435,27 @@ void Arg_CsFloatMode([[maybe_unused]] int& i, [[maybe_unused]] char* argv[])
 	}
 }
 
+uint32_t parse_u32_hex(char* str)
```

Comment:
Just change the formatting on this name and its good.

Reply:
Done, thanks